performance-timeline/po-resource-single.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE HTML>
+<meta charset=utf-8>
+<title>PerformanceObservers: resource</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="performanceobservers.js"></script>
+<h1>PerformanceObservers: resource</h1>
+<p>
+New resources will <a href="https://w3c.github.io/performance-timeline/#dfn-queue-a-performanceentry">queue a PerformanceEntry</a>.
+</p>
+<div id="log"></div>
+<script>
+  async_test(function (t) {
+    var img_location = document.location.origin
+       + "/performance-timeline/resources/square.png";
+    var observer = new PerformanceObserver(
+        t.step_func(function (entryList, obs) {
+          checkEntries(entryList.getEntriesByType("resource"),
+            [{ entryType: "resource", name: img_location}]);
+          observer.disconnect();
+          t.done();
+        })
+      );
+    observer.observe({entryTypes: ["resource"]});
+    var img = document.createElement("img");
+    img.src = img_location;
+    document.body.appendChild(img);
+    img = document.createElement("img");
+    img.src = img_location;
+    document.body.appendChild(img);
+  }, "Only one resource entry per resource");
```

Comment:
> (I'm actually very curious about how this type of testing would play out as multiple browsers implement Resource Timing on Performance Observer but I'm not yet sure we've specced how this should behave.)

We haven't.. I think this is part of "define RT in terms of Fetch". In the scenario I outlined above, with network fetch followed by an HTTP cache fetch, we have two separate calls to fetch that (I think) should emit separate RT entries. 

An open question here is how/if Fetch handles the whole "in memory cache" scenario, or if it even should.. /cc @annevk